<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btSelectPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>选择视频文件目录</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableRTSPInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>671</width>
      <height>401</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>12</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>160</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>1路</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2路</string>
     </property>
    </row>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>rtsp地址</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>视频分辨率</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>视频码流(码率)kbps</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>连接数量</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>251</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>系统信息</string>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btRTSPRun">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开启RTSP服务</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>RTSP前缀</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMideaFilePath">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>当前选择路径</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listClientInfo">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>50</y>
      <width>191</width>
      <height>321</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>新建项目</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>新建项目</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>新建项目</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="textRtspPrefix">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RTSP客户端数量</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineClientSize">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="listLocalIP">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>410</y>
      <width>191</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>390</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>本机IP列表</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAlort"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAlort">
   <property name="text">
    <string>abort(?)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
